### DISCUSSION TITLE
What is the "other street parking quest"?

### DISCUSSION THREAD

**Discussion by wielandb** [streetcomplete/StreetComplete#4092] [Q&A] [answered]
Shortly after AddStreetParking was implemented, it was disabled again. It was said that that is so that it can go ["live together with the other to be implemented street parking quest"](https://github.com/streetcomplete/StreetComplete/commit/c6ca1531e866b801c2e9d18e47dc8745cec07223). 
I would be interested to know what this other street parking quest will be/what it will ask, and of course, if any help is needed (beyond programming) that the community could already assist with in the meantime.

**Comment by matkoniecz** [comment, marked as answer]
parking:condition:*: quest - https://github.com/matkoniecz/Zazolc/issues/11

Right now you can help by reminding me that I planned to finish it already :) And you just did it :)

Feel free to ping me in a few days

**Reply by wielandb** [reply]
With the StreetParkingOverlay now in Beta, I think this is a good time for the requested ping @matkoniecz :D 

Is there already an Idea if AddStreetParkingCondition will be a quest or an overlay? I personally feel a quest would be nice, as it would only spawn when StreetParking was already added, which means the ambiguity which streetparking is appropriate would be no issue. Additionally, if it was an overlay that would mean if I want to survey an area for parking, I would have to switch overlays for every street I survey as I first record the parking situation and after that the parking condition.

**Comment by westnordost** [comment]
I think maybe the "AddStreetParking" should be a an overlay instead of a by-default-disabled quest. It is this kind of data where it would be helpful to see how the next street or just the next section of road has been tagged, especially in edge cases where one is not sure if something should still count as "on-street" parking or not.

It shouldn't be too much effort to do so, just have to come up with a reasonable color scheme.
